README.md

```diff
@@ -67,11 +70,29 @@ The feature management library supports appsettings.json as a feature flag sourc
                 }
             ]
         }
+        "FeatureW": {

```

Comment:
If we keep adding to this list it will get a bit unweildy. I'd like to see if we can limit the new features to their own sections. In this case, moving this feature you've put together to the requirement type section.